app/controllers/concerns/paginable.rb

```diff
@@ -101,8 +92,9 @@ def paginable_sort_link(sort_field)
     link_to(
       sort_link_name(sort_field),
       sort_link_url(sort_field),
-      "data-remote": true,
-      class: "paginable-action", "aria-label": "#{sort_field}"
+      class: "paginable-action",
+      data: { remote: true },
+      aria: { label: sort_field }
```

Comment:
Didn't realize you could achieve the same result by breaking up the attribute name like this. Will start using this convention going forward 👍

Reply:
neither did I, it's much easier to read though

Reply:
@briri @xsrust  very handy when you have multiple nested attributes such as:

`data: { toggle: "dropdown", target: "#my-dropdown", user_id: 1234 }`